### PR TITLE
Calculator Keyboard events use Object literal instead of ifs

### PR DESCRIPTION
**Pull request by omrihecht**
Hi Ido,
I have updated the project according to the comments.
Please take a look,
thanks

### CODE REVIEW

File: client/app/components/calculator/calculator.controller.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 
 class CalculatorController {
   constructor( $scope, $timeout, calculatorService ) {
+    this.$scope = $scope;
```
Comment (idostern) [inline client/app/components/calculator/calculator.controller.js:4]: why do you need $scope?

File: client/app/components/calculator/calculator.controller.js [hunk 2]
```diff
@@ -82,21 +79,19 @@ class CalculatorController {
     this.displayVal = this.displayVal.toString() + numVal.toString();
   }
 
-  onFnClick( calcFnBtn ){
+  onFnClick( calcFnSymbol ){
     this.equals();
-    var calcFn = this.calcFuncs[calcFnBtn]
-    this.memVal = this.displayVal + ' ' + calcFnBtn;
+    this.selectedFn = calcFnSymbol;
+    var calcFn = this.calcFuncs[calcFnSymbol];
```
Comment (idostern) [inline client/app/components/calculator/calculator.controller.js:85]: better use let then var when writing es6

**Review by idostern — APPROVED** [review]
Cool got, nice work look cleaner.

File: client/app/components/calculator/calculator.controller.js (hunk 2 above)
Comment (omrihecht) [inline client/app/components/calculator/calculator.controller.js]: I ended up not using the $scope nor the $timeout, they'v been removed.

**Comment by omrihecht** [comment]
Thanks,
I ended up not using the $scope nor the $timeout, they'v been removed.
also i changed vars to let, i'm trying to understand the difference (limited to block not function?)
have a good day

**Pull request merged**